### DISCUSSION TITLE
2006/08/21/%E3%80%8Apersonal-power%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#109] [Announcements]
# 2006/08/21/%E3%80%8Apersonal-power%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

确切的说，应该是听后感。这套《激发个人潜能》是由安东尼 罗宾（Anthiny Robbins）主讲的一套有声教程。安东尼据说是现在美国出场费最高的培训教师，他讲的课程内容基本都是关于励志、心理辅导方面的。这套长达三十天课时的《激发个人潜能》实在是长了一些，某些章节明显有拖延时间和故弄玄虚的嫌疑。但是，我不得不说，这是我所有看过的书、听过的教程中对我影响最大的一部。我的性格在这几年发生了一些明显的变化，其中变化最快的阶段就发生在聆听这套教程的一个月里。

https://blog.qizhen.xyz/2006/08/21/%E3%80%8Apersonal-power%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

**Comment by ruanqizhen** [comment]
之前博客下的留言：

如玉
2007年01月28日 17:23 编辑
這套有聲書哪裡買?

回复
永青
2006年08月28日 14:44 编辑
阮大侠啊阮大侠，你就做我减肥路上的那个教练吧。。。

回复
Qizhen
2006年08月23日 19:14 编辑
我真的非常高兴，看见有人如此认真的和我交流思想。
共勉共勉 :)

回复
竹贤
2006年08月23日 17:47 编辑
快乐不仅仅是追求的目的，更多的时候，是一种过程。如果我正在做我感兴趣的、努力后就能够做好的、社会和公众又需要的某件事，成功了我当然会快乐，而且在工作的过程我也一定是快乐着的。

回复
竹贤
2006年08月23日 15:38 编辑
无论是马斯洛的层次需求理论（生理需求、安全需求、社交需求、尊重需求和自我实现需求），还是安东尼·罗宾的激发潜能的理论，都值得一读，值得深思，对每一个期望得到发展、获得社会承认、实现自我价值的人来说，也都具有指导意义。
虽然，它们并非万能钥匙。俗话说，一把钥匙开一把锁，开启每个人心灵的钥匙应该是不同的。所以，也就有了这么多的文章、课程、书籍和理论。尽管每一种理论，同样也会有它的局限性。比如，先说马斯洛的层次需求理论。其实，每个层次的需求标准因人而异、因时而异、因地而异；简而言之，是因条件而异的。粗茶淡饭、陋居斗室可以满足有些人的生理需求；但对另一些人而言，恐怕就得有山珍海味、别墅名车、二奶小姐方能满足其生理需求。对于有些人，也许利用一切手段牟取到足够的金钱，才是自我的实现。再说说安东尼·罗宾的理论。其实，痛苦和快乐是很抽象的东西，更多的是自己的一种感觉。我们无法找到一个标准来衡量它们。还有，快乐与善、美、爱这些人类最美好的品行并不等同。人们也往往容易为获得眼前的快乐，而丢失掉了长远的幸福。我觉得，痛苦和快乐往往是并存的。甜中会有苦，苦中也一定会有甜。“福兮祸所伏，祸兮福所倚”讲的就是这个哲理。如同你、我一样，更多的读者是从这些理论中认识到了自己的不足、找到了追求的方向、获得了前进的动力。我们懂得了，工作不仅仅是为了获取得衣食住行，为了满足生理感官上的需求，而更应该谋取社会和公众的尊重，应该实现自己的生命价值。我们也因此发现了自己居然具有这么多尚未认识到的、更没有释放出来的能力和潜力，这仿佛为自己的成功增添了火箭发动机一样的动力，就等着点火发动。  多读一些这方面的书籍，有所思索，有所实践，“知识就是力量”。
共勉者

回复
清扬
2006年08月22日 11:06 编辑
我也听过,很遗憾当时没有坚持听完.确实是不错的一套教材,有时间一定要再听一次.

回复
